lib/reporting/proofing_rate_report.rb

```diff
@@ -84,55 +86,42 @@ def to_csv
       end
     end
 
+    def sub_reports
+      @sub_reports ||= by_month ? by_month_ranges : trailing_days_ranges
+    end
+
     def reports
       @reports ||= begin
-        sub_reports = [0, *DATE_INTERVALS].each_cons(2).map do |slice_end, slice_start|
-          time_range = if slice_end.zero?
-                         Range.new(
-                           (end_date - slice_start.days).beginning_of_day,
-                           (end_date - slice_end.days).end_of_day,
-                         )
-                       else
-                         Range.new(
-                           (end_date - slice_start.days).beginning_of_day,
-                           (end_date - slice_end.days).end_of_day - 1.day,
-                         )
-                       end
-          Reporting::IdentityVerificationReport.new(
-            issuers: nil, # all issuers
-            time_range: time_range,
-            cloudwatch_client: cloudwatch_client,
-          )
-        end
```

Comment:
This got replaced by `sub_reports` above. This direct code went into `trailing_days_ranges`, and I added `by_month_ranges` to do calendar-month chunks.